src/client/components/MyProfile/YourUploadedCVs/YourUploadedCVs.css

```diff
@@ -0,0 +1,117 @@
+.your-uploaded-cvs {
+  display: flex;
+  flex-direction: column;
+  width: 59vw;
+}
+
+.header-your-uploaded-cv {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: flex-start;
+  width: 100%;
+  padding-bottom: 13px;
+}
+
+.your-uploaded-cvs li {
+  list-style-type: none;
+  display: grid;
+  grid-template-columns: auto 15vw 12vw 12vw;
+  gap: 7px;
+  align-items: baseline;
+  box-shadow: 0px 0px 20px rgba(0, 0, 0, 0.05);
+  border-radius: 15px;
+  height: 60px;
+  margin: 14px 0 14px 0;
+  padding: 12px 30px 0 30px;
+  background-color: #ffffff;
+}
+.your-uploaded-cvs ul {
+  width: 100%;
+  margin: 0;
+  padding: 0;
+}
+.your-uploaded-cvs h2 {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 24px;
+  color: #000000;
+  margin-top: 0;
+}
+
+.cv-title {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 14px;
+  line-height: 16px;
+  color: #000000;
+}
+
+.cv-date {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 10px;
+  line-height: 12px;
+  color: #9a9a9a;
+}
+
+body {
```

Comment:
Styling body is very global. Is there another way to achieve this? Or maybe just remove it.

Reply:
Done!